### PR TITLE
applied theme to promo/button

### PR DESCRIPTION
**Pull request by vik753**

### CODE REVIEW

File: epam-promo/components/buttons/Button.tsx [hunk 1]
```diff
@@ -19,20 +18,26 @@ export interface ButtonMods {
...
 }
 
-export const Button = withMods<ButtonProps, ButtonMods>(
+export const Button = withMods<Omit<UUIButtonProps, "color">, ButtonMods>(
     uuiButton,
     applyButtonMods,
     (props) => ({
         dropdownIcon: systemIcons[props.size || defaultSize].foldingArrow,
         clearIcon: systemIcons[props.size || defaultSize].clear,
+        mode: mapFillToMod[props.fill],
```
Comment (dsv-sergey) [inline epam-promo/components/buttons/Button.tsx:41]: need to add default mode 'solid'

File: loveship/components/buttons/Button.scss [hunk 2]
```diff
@@ -1,155 +1,9 @@
...
-
-// Border style
-.style-round {
-    &.size-48 { border-radius: 24px; }
-    &.size-42 { border-radius: 21px; }
-    &.size-36 { border-radius: 18px; }
-    &.size-30 { border-radius: 15px; }
-    &.size-24 { border-radius: 12px; }
-    &.size-18 { border-radius: 10px; }
-}
```
Comment (dsv-sergey) [inline loveship/components/buttons/Button.scss:43]: are we refusing to support shape props?

File: loveship/components/buttons/Button.tsx [hunk 3]
```diff
@@ -14,17 +13,14 @@ export interface ButtonMods extends types.ColorMod, TextSettings {
...
 }
 
-export function applyButtonMods(mods: ButtonMods & ButtonProps) {
+export function applyButtonMods(mods: Omit<UUIButtonProps, "color"> & ButtonMods) {
     return [
-        css.root,
         css['size-' + (mods.size || defaultSize)],
-        styles['color-' + (mods.color || 'sky')],
-        css['fill-' + (mods.fill || 'solid')],
-        css['style-' + (mods.shape || 'square')],
```
Comment (dsv-sergey) [inline loveship/components/buttons/Button.tsx:23]: are we refusing to support shape props?

File: loveship/components/buttons/Button.tsx [hunk 4]
```diff
@@ -14,17 +13,14 @@ export interface ButtonMods extends types.ColorMod, TextSettings {
     fill?: types.FillStyle;
 }
 
-export function applyButtonMods(mods: ButtonMods & ButtonProps) {
+export function applyButtonMods(mods: Omit<UUIButtonProps, "color"> & ButtonMods) {
     return [
-        css.root,
         css['size-' + (mods.size || defaultSize)],
-        styles['color-' + (mods.color || 'sky')],
-        css['fill-' + (mods.fill || 'solid')],
```
Comment (dsv-sergey) [inline loveship/components/buttons/Button.tsx:22]: map fill to mode

**Comment by dsv-sergey** [comment]
Update snapshots for uui package

File: loveship/components/buttons/Button.tsx (hunk 4 above)
Comment (vik753) [reply loveship/components/buttons/Button.tsx:22]: done

File: loveship/components/buttons/Button.tsx (hunk 3 above)
Comment (vik753) [reply loveship/components/buttons/Button.tsx:23]: returned shape

File: loveship/components/buttons/Button.scss (hunk 2 above)
Comment (vik753) [reply loveship/components/buttons/Button.scss]: returned style-round

File: epam-promo/components/buttons/Button.tsx (hunk 1 above)
Comment (vik753) [reply epam-promo/components/buttons/Button.tsx]: added

**Pull request merged**